src/pages/hotel/booking/hotel-overview/HotelOverview.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { HotelDescriptionResponse } from '../../../../models/hotel/description-models/hotelDescriptionResponse'
+import './HotelOverview.scss'
+
+export type Props = {
+  hotel: HotelDescriptionResponse
+}
+
+function HotelOverview({ hotel }: Props) {
+  const imageSrc = hotel.media.mediaUrl ? hotel.media.mediaUrl[0] : undefined
+
+  return (
+    <div className="main-content-hotel">
+      <div className="hotel-content location-book">
+        <div className="image-content-book">
+          <img
+            className="book-image"
+            src={imageSrc}
+            role="presentation"
+            alt=""
+          />
+        </div>
+        <div className="hotel-details-book">
+          <h2 className="hotel-name-book">{hotel.hotelItem.hotelName}</h2>
+          <p className="hotel-address-book">
+            {hotel.hotelItem.address.streetAddress}
+          </p>
+          <div className="location-book .hotel-address-book">

```

Comment:
hotel-address-book style not applied to this div. Remove the dot (.) before hotel-address-book className.